File: CVFCS/pack1/Edge.java
package pack1;
import java.util.*;
import java.text.*;

public class Edge{
  public int id1,id2;
  public ToaDo point1,point2;
  public double doDai;
  public boolean isXoa;

  public Edge(int a,int b,ToaDo p1,ToaDo p2,double dd){
    id1 = a;
    id2 = b;
    point1 = p1;
    point2 = p2;
    doDai = dd;
    isXoa = false;
  }
}

File: CVFCS/pack1/Edge.py
from pack1.ToaDo import ToaDo

class Edge:
	def __init__(self, a, b, p1: ToaDo, p2: ToaDo, dd) -> None:
		self.id1 = a
		self.id2 = b
		self.point1 = p1
		self.point2 = p2
		self.doDai = dd
		self.isXoa = False
